### PR TITLE
fix: fix log and valid

### PR DESCRIPTION
**Pull request by ymsdgr**

### CODE REVIEW

File: controllers/WatermarkController.cs [hunk 1]
```diff
@@ -27,6 +28,21 @@ public class ImageController : ControllerBase
...
+            var logDetails = new WatermarkApiSchema.LogDetails
+            {
+                Timestamp = $"{DateTimeOffset.Now:yyyy-MM-dd HH:mm:ss.fff zzz}",
+                RequestMethod = "POST",
+                RequestBody = request,
+                RequestUrl = "/image/process",
+                StatusCode = 422,
+            };
+
+            _logger.LogInformation("invalid input while processing the image. {@LogDetails}", JsonConvert.SerializeObject(logDetails));
```
Comment (zifcrypto) [inline controllers/WatermarkController.cs]: This serialize could throw

File: controllers/WatermarkController.cs [hunk 2]
```diff
@@ -71,7 +75,7 @@ public IActionResult AddWatermark([FromBody] WatermarkApiSchema.WatermarkRequest
                 StatusCode = 500,
                 Response = ex.Message
             };
-            _logger.LogError("An error occurred while processing the image. {@LogDetails}", logDetails);
+            _logger.LogError("An error occurred while processing the image. {@LogDetails}", JsonConvert.SerializeObject(logDetails));
```
Comment (zifcrypto) [inline controllers/WatermarkController.cs]: This one too, becareful

**Review by argfoo-web3 — APPROVED** [review]
LGTM

**Review by argfoo-nft — APPROVED** [review]
LGTM

**Pull request merged**